src/packages/router/define/interfaces.js

```diff
@@ -1,5 +1,14 @@
 // @flow
 import type Router from '../index';
+import type Controller from '../../controller';
 import type { Route$opts } from '../../route';
 
-export type options = Route$opts & { router: Router };
+export type Router$route = Route$opts & {
+  router: Router;
+};
+
+export type Router$resource = {
+  path: string;
+  router: Router;
+  controllers: Map<string, Controller>;
+};
```

Comment:
The type definitions above are more consistent with the style used in the rest of the project.